ruby/config.rb

```diff
@@ -0,0 +1,18 @@
+# Load the gem
+require 'hubspot-api-client'
+require 'dotenv'
+
+Dotenv.load
+
+# Setup authorization
+Hubspot.configure do |config|
+  # Configure API key authorization: hapikey
+  config.api_key['hapikey'] = ENV['HUBSPOT_API_KEY']
+  config.error_handler = {
+    404 => {
```

Comment:
Shouldn't it be 429 instead?

Reply:
It should be the error that you're expecting to catch.

Reply:
In this example it's just 404.